csv/ParameterDefs.csv

```diff
@@ -3162,18 +3162,14 @@ ZPLSC_C,ZPLSC,zplsc_c_tilt_y,PD3743,quantity,,float32,,deg,-9999999,Tilt in Y Di
 ZPLSC_C,ZPLSC,zplsc_c_pressure,PD3746,quantity,,float32,,dBar,-9999999,Seawater Pressure,1,,,,,,,Seawater Pressure refers to the pressure exerted on a sensor in situ by the weight of the column of seawater above it. It is calculated by subtracting one standard atmosphere from the absolute pressure at the sensor to remove the weight of the atmosphere on top of the water column. The pressure at a sensor in situ provides a metric of the depth of that sensor. Units: dbar,Seawater Pressure,,
-ZPLSC,ZPLSC,zplsc_depth_range_channel_1,PD3747,array<quantity>,"[""channel_1_bins""]",float32,,m,-9999999,Channel 1 Depth Range,0,,,,,,,Channel 1 depth range of the samples,,,
-ZPLSC,ZPLSC,zplsc_frequency_channel_1,PD3748,quantity,,float32,,kHz,-9999999,Channel 1 Frequency,,,,,,,,Channel 1 Frequency,,,
-ZPLSC,ZPLSC,zplsc_values_channel_1,PD3749,array<quantity>,"[""channel_1_bins""]",float32,,dB m-1,-9999999,"Channel 1 Volume Backscattering (Sv)",0,,,,,,,"Channel 1 Volume Backscattering Strength (Sv)",,,
-ZPLSC,ZPLSC,zplsc_depth_range_channel_2,PD3750,array<quantity>,"[""channel_2_bins""]",float32,,m,-9999999,Channel 2 Depth Range,0,,,,,,,Channel 2 depth range of the samples,,,
-ZPLSC,ZPLSC,zplsc_frequency_channel_2,PD3751,quantity,,float32,,kHz,-9999999,Channel 2 Frequency,,,,,,,,"Channel 2 Frequency (If channel is available, otherwise None)",,,
-ZPLSC,ZPLSC,zplsc_values_channel_2,PD3752,array<quantity>,"[""channel_2_bins""]",float32,,dB m-1,-9999999,"Channel 2 Volume Backscattering (Sv)",0,,,,,,,"Channel 2 Volume Backscattering Strength (Sv) (If channel is available, otherwise None)",,,
-ZPLSC,ZPLSC,zplsc_depth_range_channel_3,PD3753,array<quantity>,"[""channel_3_bins""]",float32,,m,-9999999,Channel 3 Depth Range,0,,,,,,,Channel 3 depth range of the samples,,,
-ZPLSC,ZPLSC,zplsc_frequency_channel_3,PD3754,quantity,,float32,,kHz,-9999999,Channel 3 Frequency,,,,,,,,"Channel 3 Frequency (If channel is available, otherwise None)",,,
-ZPLSC,ZPLSC,zplsc_values_channel_3,PD3755,array<quantity>,"[""channel_3_bins""]",float32,,dB m-1,-9999999,"Channel 3 Volume Backscattering (Sv)",0,,,,,,,"Channel 3 Volume Backscattering Strength (Sv) (If channel is available, otherwise None)",,,
-ZPLSC,ZPLSC,zplsc_depth_range_channel_4,PD3756,array<quantity>,"[""channel_4_bins""]",float32,,m,-9999999,Channel 4 Depth Range,0,,,,,,,Channel 4 depth range of the samples,,,
-ZPLSC,ZPLSC,zplsc_frequency_channel_4,PD3757,quantity,,float32,,kHz,-9999999,Channel 4 Frequency,,,,,,,,"Channel 4 Frequency (If channel is available, otherwise None)",,,
-ZPLSC,ZPLSC,zplsc_values_channel_4,PD3758,array<quantity>,"[""channel_4_bins""]",float32,,dB m-1,-9999999,"Channel 4 Volume Backscattering (Sv)",0,,,,,,,"Channel 4 Volume Backscattering Strength (Sv) (If channel is available, otherwise None)",,,
+DEPRECATED,ZPLSC,zplsc_c_frequency_channel_1,PD3748,quantity,,uint16,,kHz,65535,Channel 1 Frequency,,,,,,,,Channel 1 frequency,,,
```

Comment:
changed from float32 to uint16 and deprecated - should only deprecate, use a new record if you want to change the data representation.

Reply:
The parameters were returned to there original name and type and then deprecated.  This is what would have been done if the deprecation had been done originally.  The new parameters with the generic names and new types were added to the bottom.